web/discord_relay/.env_example

```diff
@@ -0,0 +1,3 @@
+PORT=9103
+SENTRY_DSN="https://dc69197dffc94fcf8cc025c5f00ead45@o380324.ingest.sentry.io/5636934"
+DISCORD_TOKEN="LQx6IeNs9mtFICwrrB1xqMB2Fgl5-fvf"
```

Comment:
this isnt real right

Reply:
correct - copied an example and then regenerated it